### ISSUE TITLE
1.12.0 breaks controller API

### ISSUE THREAD

**Issue by sinamics** [zerotier/ZeroTierOne#2108]
I get 404 when trying to update a network with the latest version.

```bash
curl -i -X POST \
http://localhost:9993/controller/network/{network_id}\
-H 'Content-Type: application/json' \
-H 'X-ZT1-Auth: $Token' \
-d '{ "name": "new network name" }'
```
```bash
HTTP/1.1 404 Not Found
Content-Length: 0
Keep-Alive: timeout=5, max=5
```

## Please let us know
- What you expect to be happening.
200 OK. 

- What is actually happening?
404

- What operating system and ZeroTier version. Please try the latest ZeroTier release. 
Debian, Zerotier 1.12.0

Reproducible code:

### Create network
```bash
curl -i -X POST "http://localhost:9993/controller/network/{controller_id}" \
    -H "X-ZT1-Auth: $Token" \
    -H 'Content-Type: application/json' \
    -d '{ "name": "my awesome network" }'
```

### update network name ( Fails with 404 )
```bash
curl -i -X POST \
http://localhost:9993/controller/network/{network_id} \
    -H 'Content-Type: application/json' \
    -H 'X-ZT1-Auth: $Token' \
    -d '{ "name": "my network" }'
```

**Comment by sinamics** [comment]
You are right about the localhost resolves to ipv6, but unfortunately, the issue persists and I'm still receiving a 404 Not Found error.

```bash
curl -i -X POST \
http://127.0.0.1:9993/controller/network/{network_id} \
    -H 'Content-Type: application/json' \
    -H 'X-ZT1-Auth: $Token' \
    -d '{ "name": "my network" }'
```

**Comment by glimberg** [comment]
Fixed in 1.12.1

**Comment by knightian** [comment]
@glimberg has this regressed in 1.12.2?
